code/variables.py

```diff
@@ -171,53 +172,50 @@ def trmm_inputs(self, TRMMdirName, startDateTime, endDateTime):
         # Checks that inputs are ok
         if not TRMMdirName == "None":
             # Check if all the files exists in the TRMM directory entered
-            test, _ = iomethods.check_for_files(TRMMdirName, startDateTime, endDateTime, 3, 'hour')
+            test, _ = iomethods.check_for_files(TRMMdirName, startDateTime, endDateTime, 3, 'hour', flag=True)
```

Comment:
The flag here is always True. ie. there is not check here that governs the boolean option for flag. As such, the True option will always be taken through the code. Please explain the logic of this flag?